tests/drivers/regulator/fixed/dts/bindings/regulator-fixed.yaml

```diff
@@ -5,7 +5,7 @@ description: |
     This binding provides resources required to build and run the
     tests/drivers/regulator/fixed test in Zephyr.
 
-compatible: "test,regulator-fixed"
+compatible: "regulator-fixed"
```

Comment:
Doesn't it conflict with https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/regulator/regulator-fixed.yaml ?

Reply:
Yep, normalizing a different way now.